ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/parameter/HeaderParameterMapper.java

```diff
@@ -88,47 +79,6 @@ public HeaderParameterMapper(ParameterNode parameterNode, Map<String, String> ap
         }
     }
 
-    String getHeaderName(ParameterNode parameterNode, String defaultName) {
```

Comment:
Why did we remove this header mapping function?

Reply:
We have moved this to `CommonUtils` since the same logic is used in the `ExampleMapper` to extract the header name.